### ISSUE TITLE
Origin and robustness of code?

### ISSUE THREAD

**Issue by vincentvanhees** [R-Broadley/python-uos-activpal#1]
I am considering translating your code in [raw.py](https://github.com/R-Broadley/python-uos-activpal/blob/master/uos_activpal/io/raw.py) for reading .datx files from Python to R such that the R community can also work with this file format. I will make sure there is clear attribution to your work.

A few questions:
1. Did you develop the code based on a file format documentation? If yes, would you mind sharing it with me?
2. Are there any known bugs I should be aware of? I can try to help fix those in both the Python and R code.

**Comment by R-Broadley** [comment]
> Did you develop the code based on a file format documentation?

Yes, I was able to get the file format spec for the activPAL 3 data. I have no idea if this will work for activPAL 4 data.
I have not worked with this code or activPAL devices for quite a few years now so do not have the file to hand. I suggest reaching out to PAL Technologies.

>Are there any known bugs I should be aware of?

Not that I am aware of (but it has been years since I used it). I worked with a lot of datx files without issues, but there may be edge cases I did not hit. This was the main use I had for this code: https://salford-repository.worktribe.com/output/1358137

>I can try to help fix those in both the Python and R code

I am not actively maintaining this code, so if you would like to maintain a fork - let me know and I can add a link to the README.
